tests/test_core/test_alignment.py

```diff
@@ -244,6 +244,16 @@ def setUp(self):
         self.b = Alignment(["AAA", "AAA"])
         self.c = SequenceCollection(["AAA", "AAA"])
 
+    def test_deepcopy(self):
+        """correctly deep copy aligned objects in an alignment"""
+        path = "data/brca1_5.paml"
+        aln = load_aligned_seqs(path, array_align=False, moltype="dna")
+        aln = self.Class(aln.named_seqs)
```

Comment:
change `aln` to `seqs`.

Variable names have meaning and when `self.Class` is a `SequenceCollection`, it's not an alignment.